app/models/agents/gap_detector_agent.rb

```diff
@@ -8,6 +8,11 @@ class GapDetectorAgent < Agent
       The `value_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value of interest. If either
       this value is empty, or no Events are received, during `window_duration_in_days`, an Event will be created with
       a payload of `message`.
+
+      The `stream_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value in the event that will
```

Comment:
`value_path` provides filtering, `stream_path` splitting it.  They're not mutually exclusive. In my use cases I don't really see any need for `value_path`, but someone must have as it exists currently.

I'm opposed to combining these and you did not raise this objection in the previous pull request.